### PR TITLE
BGP link-bandwidth extended community support

### PR DESCRIPTION
**Pull request by ton31337**
Closes https://github.com/osrg/gobgp/issues/2466

### CODE REVIEW

**Comment by ton31337** [comment]
@fujita could you please take a look here?

File: api/attribute.proto [hunk 1]
```diff
@@ -400,6 +400,13 @@ message FourOctetAsSpecificExtended {
   uint32 local_admin = 4;
 }
 
+message LinkBandiwdthExtended {
+  bool is_transitive = 1;
+  uint32 sub_type = 2;
```
Comment (fujita) [inline api/attribute.proto:405]: How `is_transitive` and `sub_type` is used?

**Comment by ton31337** [comment]
Basically ￼not used at the moment. I'll drop this better. Initial idea was to use the same way as FRR (it can be send transitive), but it against RFC.

**Comment by fujita** [comment]
Understood. Let's drop them for now. Better to avoid adding unused stuff to a proto file.

File: api/attribute.proto (hunk 1 above)
Comment (ton31337) [reply api/attribute.proto]: Fixed.

**Comment by fujita** [comment]
pushed, thanks a lot!

**Pull request closed without merging**